src/tictacclojure/board.clj

```diff
@@ -28,9 +28,14 @@
     (partial replace-position-with-mark mark position)
     board))
 
-(defn moves-available
-  [board-with-move]
-  (filter integer? board-with-move))
+(defn positions-available
+  [board]
+  (filter integer? board))
+
+(defn position-available?
+  [board move]
+  (boolean
+    (some #(= move %) (positions-available board))))
```

Comment:
Or not...Turns out `contains?` is a bit misleading. 

See http://insideclojure.org/2015/01/27/clojure-tips-contains/
and https://clojuredocs.org/clojure.core/contains_q